lms/envs/common.py

```diff
@@ -85,7 +85,19 @@
     # in sync with the corresponding ones in cms/envs/common.py
     'ENABLE_DISCUSSION_SERVICE': True,
     'ENABLE_TEXTBOOK': True,
-    'ENABLE_STUDENT_NOTES': True,  # enables the student notes API and UI.
+
+    # .. feature_toggle_name: ENABLE_STUDENT_NOTES
+    # .. feature_toggle_type: feature_flag
+    # .. feature_toggle_default: True
+    # .. feature_toggle_description: Enables the Student Notes API and UI.
+    # .. feature_toggle_category: ????
+    # .. feature_toggle_use_cases: open_edx
+    # .. feature_toggle_creation_date: 2014-11-13
+    # .. feature_toggle_expiration_date: None
+    # .. feature_toggle_warnings: None
+    # .. feature_toggle_tickets: TNL-659
```

Comment:
What is the value of pointing to `TNL-659`?  The ticket simply says to add a feature flag, without much other info.

Reply:
In general we hope most Jira tickets would be more helpful than that.